packages/react-app/src/components/giveaways/participation/ActionButtons.tsx

```diff
@@ -55,6 +62,37 @@ const ManageButton = ({ giveaway }: ManageProps) => {
   );
 };
 
+const GenerateWinnersButton = ({
+  giveaway,
+  successCallback,
+  errorCallback,
+}: GenerateWinnersProps) => {
+  const generateWinners = () => {
+    FrontendApiClient.generateWinners(giveaway._id)
+      .then((_) => successCallback())
```

Comment:
Javascript doesn't mind if we don't use the arguments
```suggestion
      .then(() => successCallback())
```